apps/web/src/views/Farms/components/FarmTypesFilter.tsx

```diff
@@ -0,0 +1,139 @@
+import { useEffect, useState, useRef } from 'react'
+import { Box, Button, RocketIcon, CurrencyIcon, Flex, Text, InlineMenu, Toggle } from '@pancakeswap/uikit'
+import styled from 'styled-components'
+import { useTranslation } from '@pancakeswap/localization'
+
+interface FarmTypesFilterProps {
+  boostedOnly: boolean
+  handleSetBoostedOnly: any
```

Comment:
handleSetBoostedOnly:  (value: boolean) => void
`handleSetStableSwapOnly`, `handleSetFarmTypesEnableCount `
Add type instead of any